front/app/components/EsriMap/config.ts

```diff
@@ -28,7 +28,7 @@ export const configureMapView = (
   };
 
   // Change location of zoom widget if specified
-  if (initialData?.showZoomControls === false) {
+  if (initialData?.showZoomControls === false || isMobileOrSmaller) {
     const zoom = mapView.ui.find('zoom');
```

Comment:
Request from Rob - don't show the +/- zoom buttons on mobile, as it should be intuitive enough for scroll using two fingers.

Using a keyboard you can still zoom in/out using the plus and minus keys, so this should be fine to remove the buttons :+1:

Reply:
> Using a keyboard you can still zoom in/out using the plus and minus keys, so this should be fine to remove the buttons 👍

If I understand correctly, you are saying here that we could also remove it on desktop? But in the code it seems to only remove it for mobile?

Reply:
Sorry for the confusion - indeed we only remove it on Mobile :) :+1: